test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -177,6 +177,11 @@ private static object GetDataOfValue(ProtoCore.Mirror.MirrorData value)
                     return data;
                 }
             }
+            else if (value.Data is DesignScript.Builtin.Dictionary)
```

Comment:
so this logic was nabbed from the CommandLineRunner code - do you think you could update that code as well?

Reply:
https://github.com/DynamoDS/Dynamo/blob/520af8285bdda420e129bdf00a143273611a16dd/src/DynamoCLI/CommandLineRunner.cs#L76


not necessarily required but I know black team will be using this soon... maybe we could use this function there?